src/backend/loadContext.js

```diff
@@ -111,9 +111,11 @@ export class LoadContext {
       weightedGraph = this._overrideWeights(contractedGraph, weightsOverrides);
     }
     const plugins = this._declarations(this._pluginLoaders, project);
+    const userTypes = [].concat(...plugins.map((x) => x.userTypes));
+    const scoringNodePrefixes = userTypes.map((x) => x.prefix);
     const cred = await this._computeTask(this._compute, this._options, {
       params,
-      plugins,
+      scoringNodePrefixes,
       weightedGraph,
     });
```

Comment:
This functionality is not part of LoadContext's responsibility of composition. It's shouldn't "know" how to convert between plugin declarations and `scoringNodePrefixes`, it's just passing variables around and calling things in order.

We should either defer this to a new function `(PluginDeclaration[]) => NodeAddressT[]` which LoadContext can call, or change the signature of some existing one. For example, we're not using declarations for anything else. So `PluginLoaders.declarations()` could be replaced by `PluginLoaders.scoringNodePrefixes()`.